endpoints/decorators.py

```diff
@@ -385,14 +385,14 @@ def wrapper(namespace_name, repo_name, *args, **kwargs):
 
             if mirror is None:
                 abort(
-                    500,
+                    401,
                     "Repository %s/%s is set as a mirror but the Mirror configuration is missing."
                     % (namespace_name, repo_name),
                 )
 
             elif robot is None:
                 abort(
-                    400,
+                    401,
```

Comment:
Similar to above. I would not change this unless we have a good reason, just to avoid breaking existing clients.